src/ui/components/Notice/styles.scss

```diff
@@ -0,0 +1,97 @@
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
+@import "~ui/css/vars";
+@import "~photon-colors/colors";
+
+.Notice {
+  border-radius: $border-radius-s;
+  display: grid;
+  font-size: $font-size-s;
+  grid-gap: 4px;
+  grid-template-columns: 20px auto;
+  line-height: 15px;
+  min-height: 20px;
+  padding: 4px;
+
+  &.Notice-with-action {
+    grid-template-columns: 20px fit-content(50%) auto;
+  }
+}
+
+.Notice-icon {
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.Notice-column {
+  align-items: center;
+  display: flex;
+}
+
+.Notice-text {
+  margin: 0;
+  padding: 4px 0;
+
+  // Even though the grid has a 4px spacing,
+  // the spec calls for additional spacing
+  // in front of the action button *or* the
+  // end of the text block when there's no button.
+  @include padding-end(4px);
+}
+
+@mixin notice_button() {
+  &,
+  &.Button--small {
+    border: none;
+    font-size: $font-size-s;
+    line-height: 15px;
+    padding: 6px;
+    text-shadow: none;
+    width: max-content;
```

Comment:
This seems to be unsupported in Edge: https://caniuse.com/#search=max-content

That's not the end of the world IMO but I'd rather us be compatible. Is there any way to replicate what you're doing here and have it work with Edge?

Reply:
Ah, okay, that's fine then. 👍